integration/src/test/resources/org/mule/test/integration/exceptions/default-error-handler-lifecycle.xml

```diff
@@ -10,21 +10,33 @@
 
     <error-handler name="esA">
         <on-error-continue>
-            <test-components:lifecycle-tracker name="esAErrorHandlerTracker"/>
+            <test-components:lifecycle-tracker-check name="esAErrorHandlerTracker"/>
+        </on-error-continue>
+    </error-handler>
+
+    <error-handler name="esB">
+        <on-error-continue>
+            <test-components:lifecycle-tracker-check name="esAErrorHandlerTracker"/>
         </on-error-continue>
     </error-handler>
 
     <error-handler name="defaultEh">
         <on-error-continue>
-            <test-components:lifecycle-tracker name="defaultEhErrorHandlerTracker"/>
+            <test-components:lifecycle-tracker-check name="defaultEhErrorHandlerTracker"/>
+        </on-error-continue>
+    </error-handler>
+
+    <error-handler name="global">
+        <on-error-continue>
+            <test-components:lifecycle-tracker name="globalErrorHandlerTracker"/>
```

Comment:
what is the difference between `lifecycle-tracker` and `lifecycle-tracker-check`? whis is this one nor check?

Reply:
The `lifecycle-tracker-check` checks that the lifecycle phases are called only once, so it is necessary to use `lifecycle-tracker` in a test that starts, stops and starts again a flow.